app/models/source.rb

```diff
@@ -0,0 +1,8 @@
+class Source < ActiveRecord::Base
+  belongs_to :source_set
+  validates :aggregation, presence: true
+
+  def aggregation_uri
+    'http://dp.la/item/' + aggregation
```

Comment:
Excellent idea.  Do you think we could make this its own PR?  I am asking b/c there is another PR pending (#12) that adds the `config` gem.  Once that gem is in place, making this configurable will be much easier.  The other option would be to wait until PR#12 is merged and then rebase this branch.